project/scenes/Main.gd

```diff
@@ -1,5 +1,12 @@
 extends Node
 
+var croc_scene = preload("res://scenes/crocodile.tscn")
+var spike_scene = preload("res://scenes/spikes.tscn")
+var eagle_scene = preload("res://scenes/bird.tscn")
+var obstacle_types = [croc_scene, spike_scene]
+var obstacles : Array
```

Comment:
when doing type hints, do `obstacles: Array` rather than `obstacles : Array`. This applies to other variables that are notated like this